src/components/CurriculumComponents/CurriculumDownloadView/schema.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from "zod";
+
+const ERRORS = {
+  school: "Select school, type ‘homeschool’ or tick ‘My school isn’t listed’",
+  email: "Please enter a valid email address",
+  terms: "Accept terms and conditions to continue",
+} as const;
+
+export const schoolIdSchema = z
```

Comment:
There's a similar schema in the `resourceFormValuesSchema` in TeacherComponents/downloadAndShare.schema.ts which might be useful for consistency